src/web/wms/src/main/java/org/geoserver/wms/web/data/ExternalGraphicPanel.java

```diff
@@ -101,13 +100,8 @@ public void validate(IValidatable<String> input) {
                         } catch (URISyntaxException e1) {
                             // Unable to check if absolute
                         }
-                        if (uri != null && uri.isAbsolute()) {
+                        if (uri != null && uri.isAbsolute() && isUrl(value)) {
```

Comment:
Potentially inconsistent logic... here is ``uri.isAbsolute() && isUrl(value)`` but below in the class (getExternalGraphic) there is a ``uri.isAbsolute() || isUrl(value)``.

Reply:
will fix